repository/src/main/java/org/apache/atlas/tasks/AtlasTaskService.java

```diff
@@ -71,4 +76,31 @@ public TaskSearchResult getTasks(TaskSearchParams searchParams) throws AtlasBase
 
         return ret;
     }
+
+    @Override
+    public void retryTask(String taskGuid) throws AtlasBaseException {
+        AtlasGraphQuery query = graph.query()
```

Comment:
We had moved to ES query instead graph query to avoid getting into Tombstone issue
I suggest use ES query here instead of graph query

Reply:
I need to get the graph vertex regardless because I am updating the vertex's status. Even if I update the query and get atlas task via ES api, I need to get atlas graph vertex to update it and commit the changes.

Reply:
Task vertex is obtained from ES now.